daml-lf/scenario-interpreter/src/main/scala/com/digitalasset/daml/lf/Pretty.scala

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf
+package scenario
+
+import com.daml.lf.speedy.PartialTransaction
+
+import org.typelevel.paiges.Doc
+import org.typelevel.paiges.Doc._
+
+private[lf] object Pretty {
+
+  import speedy.Pretty._
+
+  def prettyError(err: Error, optPtx: Option[PartialTransaction] = None): Doc =
+    err match {
+      case Error.RunnerException(serror) =>
+        speedy.Pretty.prettyError(serror, optPtx)
+      case Error.Internal(msg) =>
+        text(s"CRASH: $msg ")
```

Comment:
Looks to me like `Pretty` handles `Internal`. What am I missing?

Reply:
My mistake.
I am not sure dropping `Error.Internal#toString` is a good idea, as the error can be thrown directly and then potentially improperly catch.  I think having a proper to String  method can be useful for debugging, specially because Internal Error indicate a bug somewhere.

Reply:
Then you probably also want to extend `getMessage`.